src/containers/tud/TimeUseDiaryReducer.js

```diff
@@ -71,32 +70,26 @@ export default function timeUseDiaryReducer(state :Map = INITIAL_STATE, action :
       });
     }
 
-    case downloadTudResponses.case(action.type): {
+    case downloadTudData.case(action.type): {
       const { date, dataType } = action.value;
-      return downloadTudResponses.reducer(state, action, {
-        REQUEST: () => state
-          .setIn([DOWNLOAD_TUD_RESPONSES, REQUEST_STATE, date, dataType], RequestStates.PENDING)
-          .setIn([DOWNLOAD_TUD_RESPONSES, action.id], action),
+      return downloadTudData.reducer(state, action, {
+        REQUEST: () => {
+          if (!date) {
+            return state.setIn([DOWNLOAD_ALL_DATA, REQUEST_STATE], RequestStates.PENDING);
+          }
+          return state.setIn([DOWNLOAD_TUD_DATA, REQUEST_STATE, date, dataType], RequestStates.FAILURE);
```

Comment:
Since you are merging 2 reducers into 1, you should update both `REQUEST_STATE` values to avoid getting into a weird state. 

Additionally, while this is straightforward logic, I don't think it belongs here. I think `REQUEST: () => {}` should always set `RequestStates.PENDING` and the saga should be the one responsible for deciding when to call `.failure()` and when to call `.success()`.

Reply:
Line 80 was a typo. Updated to `RequestStates.PENDING`

Reply:
> Since you are merging 2 reducers into 1, you should update both `REQUEST_STATE` values to avoid getting into a weird state.
> 
Only one of the `REQUEST_STATE` values can be updated. For example, if no date value is passed then it wouldn't make sense to update the `[DOWNLOAD_TUD_DATA, REQUEST_STATE, date, dataType]` path